apache2/re_operators.c

```diff
@@ -1025,10 +1047,6 @@ static int msre_op_rx_execute(modsec_rec *msr, msre_rule *rule, msre_var *var, c
     assert(var != NULL);
     assert(error_msg != NULL);
     msc_regex_t *regex = (msc_regex_t *)rule->op_param_data;
-    if (!regex) {
```

Comment:
This check is still missing.

Reply:
I incorrectly added the check for memory allocation problem but there's no allocation